app/src/main/res/values/styles.xml

```diff
@@ -5,9 +5,8 @@
 
         <item name="colorAccent">@color/accent</item>
         <item name="colorPrimaryDark">@color/background</item>
-        <item name="android:colorBackground">@android:color/black</item>
         <item name="android:textColor">@android:color/white</item>
-        <item name="android:textColorSecondary">@android:color/darker_gray</item>
+        <item name="android:textColorSecondary">#FAE2A4</item>
```

Comment:
great call, i realized that was dumb and fixed that a minute ago. also removed a lot of the places where i set the text color manually when it should be set by the style